lib/standalone.js

```diff
@@ -147,11 +147,12 @@ module.exports = async function standalone({
 			!postcssResult.stylelint.ignored &&
 			!postcssResult.stylelint.ruleDisableFix
 		) {
-			returnValue.output = !postcssResult.stylelint.disableWritingFix
-				? // If we're fixing, the output should be the fixed code
-				  postcssResult.root.toString(postcssResult.opts.syntax)
-				: // If the writing of the fix is disabled, the input code is returned as-is
-				  code;
+			returnValue.output =
+				!postcssResult.stylelint.disableWritingFix && postcssResult.opts
```

Comment:
[note] I've added the `postcssResult.opts` check to avoid the following error:

```
lib/standalone.js:153:36 - error TS2532: Object is possibly 'undefined'.

153        postcssResult.root.toString(postcssResult.opts.syntax)
                                       ~~~~~~~~~~~~~~~~~~
```